transitions/RotateScaleVanish.glsl

```diff
@@ -0,0 +1,40 @@
+// Author: Mark Craig
+// mrmcsoftware on github and youtube ( http://www.youtube.com/MrMcSoftware )
+// License: MIT
+
+// RotateScaleVanish Transition by Mark Craig (Copyright © 2022)
+
+uniform bool FadeInSecond; // = true
+uniform bool ReverseEffect; // = false
+uniform bool ReverseRotation; // = false
+
+#define M_PI 3.14159265358979323846
+#define _TWOPI 6.283185307179586476925286766559
+
+vec4 transition(vec2 uv)
+{
+if (progress == 0.0) { return(getFromColor(uv)); }
+else if (progress == 1.0) { return(getToColor(uv)); }
```

Comment:
if the formula of the transition is correct (moving from 'from' to 'to') you shouldn't need to do this.

Reply:
Done. I didn't have those **if**s in my own code; I put them in because I didn't know if it would pass the (automated?) tests because of the possibility of an incorrect pixel due to floating point errors, etc. But as long as it passes without the **if**s, it's better to get rid of them.